utilities/constants.py

```diff
@@ -28,84 +31,134 @@
+            DIR = f"{BASE_IMAGES_DIR}/rhel-images"
+            DEFAULT_DV_SIZE = "20Gi"
+            DEFAULT_MEMORY_SIZE = "1.5Gi"
+
+        class Windows:
+            WIN10_IMG = "win_10_uefi.qcow2"
+            WIN10_WSL2_IMG = "win_10_wsl2_uefi.qcow2"
+            WIN10_ISO_IMG = "Win10_22H2_English_x64.iso"
+            WIN2k16_IMG = "win_2k16_uefi.qcow2"
+            WIN2k19_IMG = "win_2k19_uefi.qcow2"
+            WIN2k25_IMG = "win_2k25_uefi.qcow2"
+            WIN2k19_HA_IMG = "win_2019_virtio.qcow2"
+            WIN11_IMG = "win_11.qcow2"
+            WIN11_WSL2_IMG = "win_11_wsl2.qcow2"
+            WIN11_ISO_IMG = "en-us_windows_11_business_editions_version_24h2_x64_dvd_59a1851e.iso"
+            WIN19_RAW = "win_2k19_uefi.raw"
+            WIN2022_IMG = "win_2022.qcow2"
+            WIN2022_ISO_IMG = "Windows_Server_2022_x64FRE_en-us.iso"
+            WIN2025_ISO_IMG = "windows_server_2025_x64_dvd_eval.iso"
+            DIR = f"{BASE_IMAGES_DIR}/windows-images"
+            RAW_DIR = f"{DIR}/raw_images"
+            UEFI_WIN_DIR = f"{DIR}/uefi"
+            HA_DIR = f"{DIR}/HA-images"
+            ISO_WIN10_DIR = f"{DIR}/install_iso/win10"
+            ISO_WIN11_DIR = f"{DIR}/install_iso/win11"
+            ISO_WIN2022_DIR = f"{DIR}/install_iso/win2022"
+            ISO_WIN2025_DIR = f"{DIR}/install_iso/win2025"
+            DEFAULT_DV_SIZE = "70Gi"
+            DEFAULT_MEMORY_SIZE = "8Gi"
+            DEFAULT_MEMORY_SIZE_WSL = "12Gi"
+            DEFAULT_CPU_CORES = 4
+            DEFAULT_CPU_THREADS = 2
+
+        class Fedora:
+            FEDORA41_IMG = "Fedora-Cloud-Base-Generic-41-1.4.x86_64.qcow2"
+            FEDORA_CONTAINER_IMAGE = "quay.io/openshift-cnv/qe-cnv-tests-fedora:41"
+            DISK_DEMO = "fedora-cloud-registry-disk-demo"
+            DIR = f"{BASE_IMAGES_DIR}/fedora-images"
+            DEFAULT_DV_SIZE = "10Gi"
+            DEFAULT_MEMORY_SIZE = "1Gi"
+
+        class CentOS:
+            CENTOS_STREAM_9_IMG = "CentOS-Stream-GenericCloud-9-20220107.0.x86_64.qcow2"
+            DIR = f"{BASE_IMAGES_DIR}/centos-images"
+            DEFAULT_DV_SIZE = "15Gi"
+
+        class Cdi:
+            QCOW2_IMG = "cirros-qcow2.img"
+            DIR = f"{BASE_IMAGES_DIR}/cdi-test-images"
+
+    class S390X:
+        class Cirros:
+            pass
+
+        class Rhel:
+            pass
+
+        class Windows:
+            pass
+
+        class Fedora:
+            pass
+
+        class CentOS:
+            pass
+
+        class Cdi:
+            pass
+
+    class ARM64:
+        class Cirros:
+            pass
+
+        class Rhel:
+            pass
+
+        class Windows:
+            pass
+
+        class Fedora:
+            pass
+
+        class CentOS:
+            pass
+
+        class Cdi:
+            pass
+
+
+def get_test_images_arch_class() -> Any:
+    arch = os.environ.get("OPENSHIFT_VIRTUALIZATION_TEST_IMAGES_ARCH", "x86_64")
```

Comment:
can we pick it automatically from nodes ?

Reply:
No, at this point there is no access to the cluster.